### PR TITLE
privitize attributes

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

playbooks

### CODE REVIEW

File: changelogs/fragments/private_really.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ensure private attributes stay private.
```
Comment (bmillemathias) [inline changelogs/fragments/private_really.yml]: looking to files modified I assume this is only for playbook (I did not know playbook had this), if this is the case, that could be added in the sentence.

Comment (bcoca) [reply changelogs/fragments/private_really.yml]: yep, only playbook objects have attributes .. but only very few devs actually know that, i'll update

**Comment by samdoran** [comment]
!waffling core_review

**Comment by samdoran** [comment]
!waffling needs_revision

**Pull request closed without merging**